### PR TITLE
[gadget] twig-lint

### LINKED ISSUE #51: [gadget] syntax check/lint twig files
`"asm89/twig-lint": "@stable"`

### PR DESCRIPTION
**Pull request by tolry**
add twig lint checker "asm89/twig-lint": "@stable" to prevent fatal twig errors

closes #51

### CODE REVIEW

**Comment by tolry** [comment]
ready for merge - will fix dependency to original repo, as soon as my PR is merged

Any objections @simplethings/simpspector? Who would like to merge?

**Comment by DavidBadura** [comment]
:+1:

File: src/SimpleThings/AppBundle/Gadget/TwigLintGadget.php [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    }
+
+    /**
+     * @return string
+     */
+    public function getName()
+    {
+        return self::NAME;
+    }
+}
```
Comment (larsborn) [inline src/SimpleThings/AppBundle/Gadget/TwigLintGadget.php]: I'm looking forward to the point, where we monitor SimpSpector with SimpSpector and things like the missing docblock in this file or the "wrong" alignment of assignements gets punished.

**Comment by larsborn** [comment]
without trying it out: it looks good. Let's merge it and try it out?

**Comment by tolry** [comment]
Splendid! I just added twig-lint to SimpSpector's yml file itself. Anybody but me has to press the button then :smile:

File: src/SimpleThings/AppBundle/Gadget/TwigLintGadget.php (hunk 1 above)
Comment (tolry) [inline src/SimpleThings/AppBundle/Gadget/TwigLintGadget.php]: Are you talking about the 'missing' docblock before the c'tor? Then I would like to disagree :smiley_cat: 

The c'tor has no arguments and no return value|type, therefore no docblock. Alignment is quite ugly, I will fix that - give me a minute.

Comment (tolry) [inline src/SimpleThings/AppBundle/Gadget/TwigLintGadget.php]: done

**Pull request merged**